.github/workflows/test.yaml

```diff
@@ -68,7 +68,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        feature: [auto-reconnect]
+        feature: [auto-reconnect, blog-env-postgresql]
```

Comment:
`blog-env-postgresql` is not a core feature. What is this actually testing and why?

Reply:
This is testing that query engine recovers after an error is thrown. Hence the 16 second time out on line 150 of index.spec.js. The prisma client is going through a proxy to get to the database, so the query on line 129 should fail as the port is not yet open. Queries made after the proxy is "created" and > 15s have elapsed should pass. 

The logic is from this old test:
https://github.com/prisma/prisma/blob/6b97cf3498116d7b5cac37a9a169bbd8e08685d3/src/packages/client/src/__tests__/runtime-tests/blog-env-postgresql/index.js

But as your question suggests this is really not apparent from the naming so I should change that and also remove anything that doesn't directly test that.

Reply:
That would be neat 👍